.github/workflows/pr-project-assigner.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  pull_request:
+    types: [labeled, unlabeled]
+
+jobs:
+  assign_to_project:
+    runs-on: ubuntu-latest
+    name: Assign a PR to project based on label
+    steps:
+      - name: Assign to project
+        uses: elastic/github-actions/project-assigner@v1.0.0
+        id: project_assigner
+        with:
+          issue-mappings: '[{"label": "Team:AppAch", "projectName": "kibana-app-arch", "columnId": 6173897}]'
```

Comment:
Curious, how does one find `columnId`?

Reply:
See documentation here: https://github.com/alexh97/github-actions/tree/master/project-assigner
It's a bit convoluted - you have to copy the column URL and extract it from there.